### PR TITLE
Smartling Translations Workflow

### PR DESCRIPTION
**Pull request by bencgreenberg**
## Description

Check for new Smartling completed translations on a schedule, download them and raise a pull request with the new files.

### CODE REVIEW

**Comment by bencgreenberg** [comment]
@fabianrbz, Something to consider when reviewing this is the way we currently initialize `TranslatorCoordinator` now with passing in a `frequency` integer, does that make sense? Shouldn't we dynamically generate it from either the number provided in the markdown file or in the product config, if the former is missing? This is currently unresolved and right now the Rake task will not work because it expects a manual `frequency` input, but considering it this afternoon, I think this needs to be refactored. What do you think?

File: .github/workflows/smartling_uploader.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    strategy:
+      matrix: ${{needs.check-changes.outputs.matrix}}
+    steps:
+    - name: Run rake task
+      env:
+        POSTGRES_USERNAME: postgres
+        OAS_PATH: "_open_api/api_specs/definitions"
+        FORCE_COLOR: true
+      run: |
+        RAILS_ENV=test bundle exec nexmo-developer --docs=. --rake-smartling-upload ${{matrix}}
```
Comment (bencgreenberg) [inline .github/workflows/smartling_uploader.yml:85]: @fabianrbz - In the [upload rake task](https://github.com/Nexmo/station/pull/277/files#diff-e8238f8b64f6a3f747c678954ba51f53e0a2ef22d7d494528228e881f18f81f7R56) there is an argument for frequency, which as we know is used to find files with that translation frequency either specified in the markdown metadata for that file and if not there, in the products config.

How might we specify that in the workflow here?

File: .github/workflows/smartling_uploader_15_days.yml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+name: Smartling Uploader for Files Changed within 1 Day
```
Comment (bencgreenberg) [inline .github/workflows/smartling_uploader_15_days.yml:1]:
```suggestion
name: Smartling Uploader for Files Changed within 15 Days
```

File: .github/workflows/smartling_uploader_30_days.yml [hunk 3]
```diff
@@ -0,0 +1,88 @@
+name: Smartling Uploader for Files Changed within 1 Day
```
Comment (bencgreenberg) [inline .github/workflows/smartling_uploader_30_days.yml:1]:
```suggestion
name: Smartling Uploader for Files Changed within 30 Days
```

File: .github/workflows/smartling_uploader_180_days.yml [hunk 4]
```diff
@@ -0,0 +1,83 @@
+name: Smartling Uploader for Files Changed within 180 Days
+on:
+  push:
+    branches:
```
Comment (fabianrbz) [inline .github/workflows/smartling_uploader_180_days.yml:4]: @bencgreenberg this should run on a schedule, every 180 days

File: .github/workflows/smartling_uploader_5_days.yml [hunk 5]
```diff
@@ -0,0 +1,83 @@
+name: Smartling Uploader for Files Changed within 5 Days
+on:
+  push:
+    branches:
+      - master
```
Comment (fabianrbz) [inline .github/workflows/smartling_uploader_5_days.yml:5]: same here but 5 days

**Review by fabianrbz — CHANGES_REQUESTED** [review]

File: .github/workflows/smartling_uploader_5_days.yml (hunk 5 above)
Comment (bencgreenberg) [reply .github/workflows/smartling_uploader_5_days.yml:7]: Done

File: .github/workflows/smartling_uploader_180_days.yml (hunk 4 above)
Comment (bencgreenberg) [reply .github/workflows/smartling_uploader_180_days.yml:6]: Done

**Review by fabianrbz — APPROVED** [review]

**Pull request merged**